tests/graphql/auth.graphql

```diff
@@ -0,0 +1,17 @@
+#> server-sdl
+schema @server(auth: [{basic: {file: "tests/server/config/htpasswd.txt"}, id: "basic"}, {id: "jwt", jwt: {jwks: {file: "tests/server/config/jwks.json"}}}]) @upstream {
```

Comment:
If we split this into two separate files, we will be able to keep prettier happy.

Reply:
I already have two separate tests for jwt and basic. This combined tests is used to test that several providers could be defined